### PR TITLE
Add ListViewInsertionMark ComCtl32 tests 

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Add ListViewInsertionMark ComCtl32 tests
- Cleanup ListViewInsertionMark interop (extracted from #2459 with cleanup)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2556)

### CODE REVIEW

**Comment by hughbe** [comment]
Blocked on the same thing as #2555 and #2453

**Comment by RussKie** [comment]
Please rebase

**Comment by hughbe** [comment]
Now a test only changed

File: src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewInsertionMarkTests.cs [hunk 1]
```diff
@@ -4,65 +4,187 @@
...
-            // Set with color.
-            insertionMark.Color = Color.Blue;
-            insertionMark.AppearsAfterItem = true;
-            Assert.True(insertionMark.AppearsAfterItem);
+                // Set same.
+                Assert.NotEqual(IntPtr.Zero, control.Handle);
+                control.InsertionMark.AppearsAfterItem = false;
+                var insertMark = new ComCtl32.LVINSERTMARK
+                {
+                    cbSize = (uint)Marshal.SizeOf<ComCtl32.LVINSERTMARK>()
```
Comment (RussKie) [inline src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewInsertionMarkTests.cs]: This should be 
```suggestion
                    cbSize = (uint)sizeof(ComCtl32.LVINSERTMARK)
```

Comment (hughbe) [reply src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewInsertionMarkTests.cs]: Fixed

**Pull request merged**